PublicMapping/districtbuilder
# Make org project templates active or inactive

## Overview

- Adds `isActive` field to Project Template entity
- Only display active templates on Org screen

### Checklist



### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

- Create an organization with a project template using the `update-organization` command
- Duplicate the organization's project template in the psql shell
- Update the value of `isActive` on the copied record to be false
- Reload the organization screen; expect that the second template is no longer displayed on the screen

Closes #597
